jwst/extract_2d/grisms.py

```diff
@@ -373,7 +373,7 @@ def extract_grism_objects(input_model,
                 ext_err = input_model.err[ymin: ymax + 1, xmin: xmax + 1].copy()
                 ext_dq = input_model.dq[ymin: ymax + 1, xmin: xmax + 1].copy()
 
-                tr.bounding_box = util.bounding_box_from_shape(ext_data.shape)
+                tr.bounding_box = util.bounding_box_for_transform(ext_data.shape)
```

Comment:
Is `input_model` here always an `ImageModel`? Can it be a `CubeModel` (for tso observations??

Reply:
No, there's a separate function just for TSO extractions because they dont require a catalog
https://github.com/spacetelescope/jwst/blob/master/jwst/extract_2d/grisms.py#L18